pubspec.yaml

```diff
@@ -4,7 +4,7 @@ author: Dart Team <misc@dartlang.org>
 description: Obtain Access credentials for Google services using OAuth 2.0
 homepage: https://github.com/dart-lang/googleapis_auth
 environment:
-  sdk: '>=2.0.0 <3.0.0'
+  sdk: '>=2.1.0 <3.0.0'
```

Comment:
Bump the version to `0.2.13-dev` here and make a note of this change in the changelog

Just to follow our recommendations...

Reply:
Seems like 0.2.12+1 could be a more consistent next version? Otherwise done!